### PR TITLE
RSDEV-1514: Move all legacy pagination to Bootstrap-based tag

### PR DESCRIPTION
**Pull request by tchoi-rs**
## Description ##
Move all legacy pagination to Bootstrap-based tag, and removes the legacy, plaintext pagination.

Fixes RSDEV-1514

### CODE REVIEW

**Comment by tchoi-rs** [comment]
@pullfrogai

File: src/main/webapp/WEB-INF/tags/paginate.tag [hunk 1]
```diff
@@ -11,24 +11,31 @@
...
+  <c:forEach var="page" items="${paginationList}">
+    <c:set var="href" value="${page.link}"/>
+    <c:if test="${empty isRegularLink or isRegularLink eq 'false'}">
+      <c:set var="href" value="#"/>
+    </c:if>
+
+    <c:choose>
       <c:when test="${page.link=='#'}">
-      	<span style="color: #666;">${page.name}</span>
+        <li class="active"><a class="${page.className}" href="#">${page.name}</a></li>
```
[the remark this answers is not in the document]
Comment (tyteen4a03) [reply src/main/webapp/WEB-INF/tags/paginate.tag]: Fixed in 69578cf4b. The active page now renders as an active list item containing a span with aria-current=page, so it receives no page-link handler. Verified that page 2 loads once with HTTP 200 and activating the current span issues no request or error.

File: src/main/webapp/WEB-INF/tags/paginate.tag [hunk 2]
```diff
@@ -11,24 +11,31 @@
 
 <%-- This sets the links to previous /next pages  assuming an Ajax
  based handling to parse the link out of the ID--%>
-<c:forEach var="page" items="${paginationList}"> 
-  <c:set var="href" value="${page.link}"/>
-  <c:if test="${empty isRegularLink   or isRegularLink eq 'false'}">
-   <c:set var="href" value="#"/>
-  </c:if>
-	<c:choose>
+<ul class="pagination new">
```
[the remark this answers is not in the document]
Comment (tyteen4a03) [reply src/main/webapp/WEB-INF/tags/paginate.tag:14]: Fixed in 69578cf4b. The shared dashboard container now retains bootstrap-custom-flat while dashboard.js swaps only the list-specific classes. Verified that notification pagination renders horizontally in the dev stack.

File: src/main/webapp/WEB-INF/tags/paginate.tag [hunk 3]
```diff
@@ -11,24 +11,31 @@
...
+  <c:forEach var="page" items="${paginationList}">
+    <c:set var="href" value="${page.link}"/>
+    <c:if test="${empty isRegularLink or isRegularLink eq 'false'}">
+      <c:set var="href" value="#"/>
+    </c:if>
+
+    <c:choose>
       <c:when test="${page.link=='#'}">
-      	<span style="color: #666;">${page.name}</span>
+        <li class="active"><span aria-current="page">${page.name}</span></li>
```
Comment (rs-nicof) [inline src/main/webapp/WEB-INF/tags/paginate.tag:17]: > Written by Opus 5, an AI coding agent.

The current page is now a `<span>`, but RSpace's own highlight rules in `rs-global.css` (`.pagination.new li a` and `li.active a`) only target `<a>`. Inside a Bootstrap-scoped container the current page falls back to Bootstrap's styling, so it renders a different blue and a different box size from the numbers beside it; outside one it gets no highlight at all. The "(current page)" tooltip also stops appearing, since that code only iterates `<a>` elements.

Widening those two selectors to cover `span` restores it.

File: src/main/webapp/WEB-INF/pages/dashboard/dashboard.jsp [hunk 4]
```diff
@@ -53,7 +53,7 @@
     <a class="dashboardOptions" id="createCalendarEntryDlgLink" href="#"><spring:message code="dashboard.new.calendar.entry"/>  </a>
     <jsp:include page="../workspace/calendarDialog.jsp" />
 	</div>
-	<div class="dashboardContainer notificationList"><jsp:include page="notifications_ajax.jsp"></jsp:include></div>
+	<div class="dashboardContainer notificationList bootstrap-custom-flat"><jsp:include page="notifications_ajax.jsp"></jsp:include></div>
```
Comment (rs-nicof) [inline src/main/webapp/WEB-INF/pages/dashboard/dashboard.jsp:56]: > Written by Opus 5, an AI coding agent.

This covers the Dashboard's own container, but the same notifications and messages fragments are also loaded into the two jQuery UI dialogs declared in `common/header.jsp`, which have no `bootstrap-custom-flat` ancestor. Those dialogs open from the header on every page, so their pagination keeps the browser's default 40px list indent, loses its button borders, and shows no current-page highlight.

Adding the class to those two divs closes it.

File: src/main/webapp/WEB-INF/tags/createFromFormDlg.tag [hunk 5]
```diff
@@ -10,26 +10,7 @@
...
-        }
-       .pagin  a:hover {
-            color: #cc0000;
-            text-decoration: none;
-            font-size:1em;
-        }
-</style>
-	<span class="pagin">
-<axt:paginate paginationList="${formsForCreateMenuPagination}"></axt:paginate></span>
+<axt:paginate paginationList="${formsForCreateMenuPagination}"></axt:paginate>
```
Comment (rs-nicof) [inline src/main/webapp/WEB-INF/tags/createFromFormDlg.tag]: > Written by Opus 5, an AI coding agent.

Same gap here: this renders inside the create-from-form jQuery UI dialog, which has no `bootstrap-custom-flat` ancestor, and the inline styles that used to colour these links were removed in this hunk. The picker's pagination will pick up the default list indent and lose its button styling.

Wrapping the tag in a `bootstrap-custom-flat` div, or adding it to the dialog, matches what you did for the dashboard.

File: src/main/webapp/WEB-INF/tags/paginate.tag [hunk 6]
```diff
@@ -11,24 +11,31 @@
...
  based handling to parse the link out of the ID--%>
-<c:forEach var="page" items="${paginationList}"> 
-  <c:set var="href" value="${page.link}"/>
-  <c:if test="${empty isRegularLink   or isRegularLink eq 'false'}">
-   <c:set var="href" value="#"/>
-  </c:if>
-	<c:choose>
+<ul class="pagination new">
+  <c:forEach var="page" items="${paginationList}">
+    <c:set var="href" value="${page.link}"/>
```
Comment (rs-nicof) [inline src/main/webapp/WEB-INF/tags/paginate.tag]: > Written by Opus 5, an AI coding agent.

`isRegularLink` isn't passed by any caller in the repo, so this always falls through to `href="#"` on the next line. Worth deleting the attribute and these three lines while the tag is being rewritten.

**Review by rs-nicof — COMMENTED** [review]
> Written by Opus 5, an AI coding agent.

Automated multi-reviewer pass over the pagination consolidation, covering markup and accessibility, CSS contracts, security and i18n. Four things worth a look. No security issues found.

File: src/main/webapp/WEB-INF/tags/paginate.tag (hunk 3 above)
Comment (tyteen4a03) [reply src/main/webapp/WEB-INF/tags/paginate.tag]: Fixed in 6dbf1cc7c. The shared pagination CSS and tooltip helper now include the active span. Playwright confirmed the active page has the same 32.5 px width and padding as adjacent links, uses the expected blue/white styling, and exposes the current-page title and aria-current attribute.

File: src/main/webapp/WEB-INF/tags/paginate.tag (hunk 6 above)
Comment (tyteen4a03) [reply src/main/webapp/WEB-INF/tags/paginate.tag]: Fixed in 6dbf1cc7c. Removed the unused isRegularLink tag attribute and branch after confirming there are no callers, and made the AJAX-only href behavior explicit.

File: src/main/webapp/WEB-INF/tags/createFromFormDlg.tag (hunk 5 above)
Comment (tyteen4a03) [reply src/main/webapp/WEB-INF/tags/createFromFormDlg.tag]: Fixed in 6dbf1cc7c by wrapping this pager in bootstrap-custom-flat. Playwright exercised the real controller fragment with a two-item page size: the pager rendered horizontally, page 2 loaded once with HTTP 200, and the second set of forms replaced the first.

File: src/main/webapp/WEB-INF/pages/dashboard/dashboard.jsp (hunk 4 above)
Comment (tyteen4a03) [reply src/main/webapp/WEB-INF/pages/dashboard/dashboard.jsp]: Fixed in 6dbf1cc7c. Both header dialog content containers now retain bootstrap-custom-flat. Playwright confirmed notification pagination renders horizontally and page 2 loads once with HTTP 200. The message container has the same scope; its existing /dashboard/ajax/allMessages 500 still prevents content-level verification and is unrelated to this change.

**Review by rs-nicof — APPROVED** [review]
